actions/create-update-pr/action.yml

```diff
@@ -0,0 +1,39 @@
+name: "Label new pull requests"
+description: "Add a label when a new PR is opened."
+author: "Leonardo Montini"

```

Comment:
might want to add a reference (e.g. based on https://...) to where you got this from, but once that is added I think you can remove the author here.

Did you make changes to it or could we use this action directly?

Reply:
That just the template, should not have been included or should have been changed

Reply:
Removed